modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -18,7 +18,7 @@ exports.setupRoutes = (app, loginRoutes) ->
 	app.get '/cmpdReg/parentAnnotations', loginRoutes.ensureAuthenticated, exports.getBasicCmpdReg
 	app.get '/cmpdReg/fileTypes', loginRoutes.ensureAuthenticated, exports.getBasicCmpdReg
 	app.get '/cmpdReg/projects', loginRoutes.ensureAuthenticated, exports.getAuthorizedCmpdRegProjects
-	app.get '/cmpdReg/vendors', loginRoutes.ensureAuthenticated, exports.getBasicCmpdReg
```

Comment:
Thanks. I switched it when I hit a 500 error on that route and then it worked.  I didn't end up digging into it until I saw your comment.  I was missing an @Autowired in roo.  I just fixed roo and returned this back to the original line to avoid any issues.